src/App.jsx

```diff
@@ -1,15 +1,13 @@
 import React from 'react';
 import './App.scss';
+import { Sum } from './components/Sum';
 
 const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove comment lines :) */}
+    <Sum a={2} b={3} />
+    <Sum a={5} b={-5} />
+    <Sum a={10} />
+    <Sum b={5} />
```

Comment:
You forgot about last `<p>` without parameters
<img width="158" alt="image" src="https://user-images.githubusercontent.com/99096412/165595557-e31f6e30-9873-480b-a37f-5b12f5f795c8.png">